network_importer/adapters/nautobot_api/inventory.py

```diff
@@ -44,7 +43,9 @@ def __init__(
 
         # Build Filter based on inventory_settings filter and on limit
         self.filter_parameters = {}
-        build_filter_params(self.settings.filter.split((",")), self.filter_parameters)
+        if self.settings.filter is not None:
+            build_filter_params(self.settings.filter.split((",")), self.filter_parameters)
+
```

Comment:
This was done as build_filter_params was failing when it was None. This is something that I need to add to the NetBox inventory as well.